scripts/build_jacocorunner/build_jacocorunner_bazel_5.0+.sh

```diff
@@ -108,7 +114,9 @@ mvn clean install
 (
 cd $bazel_repo
 git remote update
-git checkout tags/$bazel_tag
+git checkout $bazel_branch
+
+echo "$bazel_version" > .bazelversion
```

Comment:
:+1:

Reply:
This is fixing Bazel autodownloading version 6+ when building JacocoRunner, that breaks that build...